downstairs/src/region.rs

```diff
@@ -189,8 +189,8 @@ impl Inner {
             stmt.query_map(params![block, block + count - 1], |row| {
                 let block_index: u64 = row.get(0)?;
                 let hash: i64 = row.get(1)?;
-                let nonce: Option<Vec<u8>> = row.get(2)?;
-                let tag: Option<Vec<u8>> = row.get(3)?;
+                let nonce: Option<[u8; 12]> = row.get(2)?;
```

Comment:
Is this a database format change?  Like, can we read a region created with
the old code?

Reply:
It is backwards compatible, unless someone has deliberately messed up the metadb (which would also break `main`).

To test this, I created regions on `main` with `dsc create  --cleanup --ds-bin=target/release/crucible-downstairs --region-dir=tmp`, then ran `crutest` on `main` to populate some data.  After switching back to this branch, `dsc start` and `crutest` both worked fine.